extensions/wikia/Chat2/tests/ChatWidgetTest.php

```diff
@@ -111,6 +112,7 @@ public function testGeTemplateVarsDataProvider() {
 				'expected' => [
 					'blankImgUrl' => 'www.url.com',
 					'buttonText' => 'message',
+					'buttonIcon' => '<svg></svg>',
 					'guidelinesText' => 'message',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "guidelinesText" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)